features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Create/GraphQL/APICreateGrpahQL.jsx

```diff
@@ -0,0 +1,388 @@
+
+    handleBack = () => {
+        this.setState(state => ({
+            activeStep: state.activeStep - 1,
+        }));
+    };
+
+    handleReset = () => {
+        this.setState({
+            activeStep: 0,
+        });
+    };
+
+    /**
+     *
+     *
+     * @returns
+     * @memberof APICreateGraphQL
+     */
+    render() {
+        const { classes } = this.props;
+        const steps = getSteps();
+        const {
+            doValidate, activeStep, graphQLBean, api, valid, loading,
+        } = this.state;
+        const uploadMethod = 'file';
+        const provideGraphQLProps = {
+            [uploadMethod]: graphQLBean[uploadMethod],
+            updateGraphQLBean: this.updateGraphQLBean,
+            validate: doValidate,
+            valid,
+            updateFileErrors: this.updateFileErrors,
+        };
+        if (loading) {
+            return <Progress />;
+        }
+        return (
+            <Grid container spacing={24} className={classes.root}>
+                <Grid item xs={12} xl={6}>
+                    <div className={classes.titleWrapper}>
+                        <Typography variant='h4' align='left' className={classes.mainTitle}>
+                            <FormattedMessage
+                                id='Apis.GraphQL.APICreateGraphQL.using.SDL'
+                                defaultMessage='Design a new GraphQL API using SDL'
+                            />
+                        </Typography>
+                    </div>
+                    <Stepper activeStep={activeStep} className={classes.stepper}>
+                        {steps.map((label) => {
+                            const props = {};
+                            const labelProps = {};
+
+                            return (
+                                <Step key={label} {...props}>
+                                    <StepLabel {...labelProps}>{label}</StepLabel>
+                                </Step>
+                            );
+                        })}
+                    </Stepper>
+                    <div>
+                        {activeStep === 0 && (
+                            <ProvideGraphQL
+                                {...provideGraphQLProps}
+                            />
+                        )}
+                        {activeStep === 1 && (
+                            <React.Fragment>
+                                <APIInputForm api={api} handleInputChange={this.updateApiInputs} valid={valid} />
+                            </React.Fragment>
+                        )}
+                    </div>
+                    <div>
+                        <div>
+                            <div>
+                                <Button
+                                    disabled={activeStep === 0}
+                                    onClick={this.handleBack}
+                                    className={classes.button}
+                                >
+                                Back
```

Comment:
Need to localize

Reply:
This file is no longer used as the code had been refactored with react hooks
https://github.com/wso2/carbon-apimgt/pull/6944/files
Therefore, we don't want to add localization at here